.github/workflows/sonarcloud.yml

```
@@ -9,35 +9,18 @@
 
-      - name: SonarCloud Scan
-        uses: SonarSource/sonarcloud-github-action@v2
-        with:
-          projectBaseDir: backend
+      - name: SonarQube Scan
+        uses: SonarSource/sonarqube-scan-action@v5
         env:
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+        with:
+          projectBaseDir: backend
+         
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)